### ISSUE TITLE
failure in training

### ISSUE THREAD

**Issue by whulc** [MichalBusta/DeepTextSpotter#35]
I try to reproduce your experiment on icdar dataset, but meet the following problem:  
![qq 201805082215082](https://user-images.githubusercontent.com/33627948/39770766-8db5b458-5322-11e8-84ec-e3b1d239224a.jpg)
![qq 201805082247052](https://user-images.githubusercontent.com/33627948/39770767-8e078904-5322-11e8-99ea-137e94846fff.jpg)
![qq 201805082247292](https://user-images.githubusercontent.com/33627948/39770769-8e59cd40-5322-11e8-9b6e-3c09c4686acc.jpg)
    It seems that the data is not imported successfully, but I think the data is organized in the right way as required.

**Comment by MichalBusta** [comment]
maybe same issue as https://github.com/MichalBusta/DeepTextSpotter/issues/32 ?

**Comment by whulc** [comment]
Thank you,  but I come up with another problem, I use the model you provide as pre-trained model, but in the training process, the predict result is right but it get a heavy loss, thus change the model vastly and the predict result get worse and worse. Here is a snapshot of the training process.
 
![qq 20180514225528](https://user-images.githubusercontent.com/33627948/40005542-c06dfaa2-57ca-11e8-8478-c004068e9612.jpg)

**Comment by phaertinger** [comment]
Try `index = 4` in train.py, line 46 (https://github.com/MichalBusta/DeepTextSpotter/blob/master/train.py#L46).

**Comment by whulc** [comment]
Thank you for your suggestion, but it doesn't work, the problem remains.
![tim 20180517171557](https://user-images.githubusercontent.com/33627948/40168610-76ab2088-59f6-11e8-92e7-24be79d2ff79.jpg)

**Comment by 19931991** [comment]
hello , I research ocr recently , can we make a friend by qq:1323369151，thank you very much!

**Comment by 19931991** [comment]
hello , I research ocr recently , can we make a friend by qq:1323369151，thank you very much!@whulc

**Comment by MichalBusta** [comment]
Ref: #53
the problem:
 - we are using momentum optimizer (and momentum is in this case important)
solution is to pretrain stand-alone OCR on crops, and use model with optimizer parameters

**Comment by mohammadSadeq** [comment]
@MichalBusta @whulc 
I'm having the same issue(that the data is not imported successfully), how did you fix the problem after referencing issue #32??
Please do specify the things that you were doing wrong and what did you do to fix them
Thank you

**Comment by mohammadSadeq** [comment]
@whulc Please help me with your first issue and how did you fix it,
You will be a true life saver mate :D

**Comment by MichalBusta** [comment]
> @MichalBusta @whulc
> I'm having the same issue(that the data is not imported successfully), how did you fix the problem after referencing issue #32??

the issue ref is #53 

> Please do specify the things that you were doing wrong and what did you do to fix them
> Thank you
and again:  solution is to pretrain stand-alone OCR on crops, and use model with optimizer parameters :)

**Comment by mohammadSadeq** [comment]
@MichalBusta  I Believe you thought I was talking about the second problem, that is not my case,
I'm getting this issue 
> 
> 
> ```
> I try to reproduce your experiment on icdar dataset, but meet the following problem:  
> ```
> 
> ![qq 201805082215082](https://user-images.githubusercontent.com/33627948/39770766-8db5b458-5322-11e8-84ec-e3b1d239224a.jpg)
> ![qq 201805082247052](https://user-images.githubusercontent.com/33627948/39770767-8e078904-5322-11e8-99ea-137e94846fff.jpg)
> ![qq 201805082247292](https://user-images.githubusercontent.com/33627948/39770769-8e59cd40-5322-11e8-9b6e-3c09c4686acc.jpg)
> It seems that the data is not imported successfully, but I think the data is organized in the right way as required.

**Comment by lfdeep** [comment]
> ```
> I try to reproduce your experiment on icdar dataset, but meet the following problem:  
> ```
> ![qq 201805082215082](https://user-images.githubusercontent.com/33627948/39770766-8db5b458-5322-11e8-84ec-e3b1d239224a.jpg)
> ![qq 201805082247052](https://user-images.githubusercontent.com/33627948/39770767-8e078904-5322-11e8-99ea-137e94846fff.jpg)
> ![qq 201805082247292](https://user-images.githubusercontent.com/33627948/39770769-8e59cd40-5322-11e8-9b6e-3c09c4686acc.jpg)
> It seems that the data is not imported successfully, but I think the data is organized in the right way as required.

Hello, i meet the same problem,How did you solve it?thanks
